src/Process-Data/Create_Games.py

```diff
@@ -1,29 +1,37 @@
 import os
+import sqlite3
 import numpy as np
 import pandas as pd
+import sys
 from tqdm import tqdm
-from src.Utils.Dictionaries import team_index_07, team_index_08, team_index_12, team_index_13, team_index_14
+sys.path.insert(1, os.path.join(sys.path[0], '..'))
+from Utils.Dictionaries import team_index_07, team_index_08, team_index_12, team_index_13, team_index_14, team_index_current
+
+# season_array = ["2007-08", "2008-09", "2009-10", "2010-11", "2011-12", "2012-13", "2013-14", "2014-15", "2015-16",
+#                 "2016-17", "2017-18", "2018-19", "2019-20", "2020-21", "2021-22", "2022-23"]
+season_array = ["2015-16", "2016-17", "2017-18", "2018-19", "2019-20", "2020-21", "2021-22", "2022-23"]
```

Comment:
I did this because I remembered in your notes this comment:

>  we achieved the highest levels of validation accuracy when the training dataset started from the 2012 − 2013 season

I'll change it to go from 2012-13 instead of 2015-16.